XamarinCommunityToolkit/Views/SnackBar/Options/ToastOptions.shared.cs

```diff
@@ -23,7 +23,7 @@ public class ToastOptions
 		/// </summary>
 		public Color BackgroundColor { get; set; } = DefaultBackgroundColor;
 
-		public static Color DefaultBackgroundColor { get; set; } = Color.Default;
+		public static Color DefaultBackgroundColor { get; set; } = Color.White;
```

Comment:
it is not related, but fix UWP default setting to the Snackbar. It was black text on the black view.